### PR TITLE
created models and static pages

### PR DESCRIPTION
**Pull request by TheTrung**
Changes:  
1. created static pages: home, categories, word list
2. created models: user, activity, lesson, word, answer, question, lessonword(index)
3. modified Gemfile

### CODE REVIEW

File: app/assets/javascripts/static_pages.coffee [hunk 1]
```diff
@@ -0,0 +1,3 @@
+# Place all the behaviors and hooks related to the matching controller here.
```
Comment (chuanhtuan) [inline app/assets/javascripts/static_pages.coffee]: remove file

File: app/assets/stylesheets/static_pages.css.scss [hunk 2]
```diff
@@ -0,0 +1,3 @@
+// Place all the styles related to the StaticPages controller here.
```
Comment (chuanhtuan) [inline app/assets/stylesheets/static_pages.css.scss]: remove file

File: app/controllers/static_pages_controller.rb [hunk 3]
```diff
@@ -0,0 +1,17 @@
+class StaticPagesController < ApplicationController
+  def Home
+   # if logged_in?
```
Comment (chuanhtuan) [inline app/controllers/static_pages_controller.rb]: remove all remark code

File: app/controllers/static_pages_controller.rb [hunk 4]
```diff
@@ -0,0 +1,17 @@
...
+   #    @micropost = current_user.microposts.build 
+   #    @feed_items = current_user.feed.paginate(page: params[:page]) 
+   #  else
+   #    @micropost = nil
+   #    @feed_items = nil
+   #  end
+  end
+
+  def WordList
```
Comment (chuanhtuan) [inline app/controllers/static_pages_controller.rb]: remove

File: app/controllers/static_pages_controller.rb [hunk 5]
```diff
@@ -0,0 +1,17 @@
...
+  end
+
+  def Categories
```
Comment (chuanhtuan) [inline app/controllers/static_pages_controller.rb]: remove

File: app/models/activity.rb [hunk 6]
```diff
@@ -0,0 +1,11 @@
+class Activity < ActiveRecord::Base
+	#belongs_to :user
+	belongs_to :learnedwords, class_name: "User"
```
Comment (chuanhtuan) [inline app/models/activity.rb]: indent 2

File: app/models/answer.rb [hunk 7]
```diff
@@ -0,0 +1,4 @@
+class Answer < ActiveRecord::Base
+	belongs_to :word
```
Comment (chuanhtuan) [inline app/models/answer.rb]: indent 2

File: config/routes.rb [hunk 8]
```diff
@@ -1,4 +1,11 @@
 Rails.application.routes.draw do
+
+  root 'static_pages#Home'
```
Comment (chuanhtuan) [inline config/routes.rb]: root 'static_pages#home'

**Comment by TheTrung** [comment]
Fixed all comments.

**Pull request closed without merging**